social_platform/serializers.py

```diff
@@ -86,12 +87,18 @@ class PostDeleteSerializer(serializers.Serializer):
     username = serializers.CharField(required=True)
     password = serializers.CharField(required=True)
 class LikeSerializer(serializers.Serializer):
-    post_id = serializers.IntegerField(required=True)
     user_id = serializers.IntegerField(required=True)
+    post_id = serializers.IntegerField(required=True)
 class CommentSerializer(serializers.Serializer):
     post_id = serializers.IntegerField(required=True)
     user_id = serializers.IntegerField(required=True)
     comment=serializers.CharField(required=True)
 class ViewLikedSerializer(serializers.Serializer):
     user_id = serializers.IntegerField(required=True)
+class FollowSerializer(serializers.Serializer):
+    follower_id=serializers.IntegerField(required=True)
+    following_id=serializers.IntegerField(required=True)
+class ViewFollowingPostSerializer(serializers.Serializer):
```

Comment:
doesn't require too many serializers for every particular fields.
Remove all unnecessary serializers.